ChangeLog

```diff
@@ -1,5 +1,139 @@
+  (contributed by @app/dependabot)
+
+- Autobuild: Hardened detection of QT 5 builds for macOS and iOS (#3057).
+  (contributed by @ann0see)
+
+- Build: Windows 64bit releases use Qt 6.4.3 now. Jamulus will from now on only support Windows 10 and above (#2300).
+  (contributed by @hoffie)
+
+- Translation: Document translation process shift to Weblate. Translators can from now one use Weblate for translations (#2880).
+  (contributed by @ann0see)
+
+- Build: Updated bundled Qt6 to version 6.4.0 (#2892).
+  (contributed by @app/github-actions)
+
+- Refactor: use `size_t` for vector and array indexes that must not be negative (#2893).
+  (contributed by @pljones)
+
+- Internal: Include HTML in strDirectoryTypeWT to make translation easier (#2894).
+  (contributed by @pljones)
+
+- Android: Fixed a crash on session startup that occurs on many devices (#2905).
+  (contributed by @danryu)
+
+- New command line switch (--jsonrpcbindip) to specify the bind address for the RPC server (#2917).
+  (contributed by @Rob-NY)
+
+- jamulusserver/getClients method expanded to include all elements of a user profile as well as the number of active client connections (#2918).
+  (contributed by @Rob-NY)
+
+- Translations updated from Hosted Weblate (#2921).
+  (contributed by @weblate)
+
+- Translations updated from Hosted Weblate (#2923).
+  (contributed by @weblate)
+
+- Internal: Renamed English translation file of windows installer to `en_GB` (#2929).
+  (contributed by @ann0see)
+
+- Client: Added SRV-based virtual hosting support (#2930).
+  (contributed by @hoffie)
+
+- Server: ISO country codes can now be supplied in upper or lower case (#2934).
+  (contributed by @Rob-NY)
+
+- Build: Bump maxim-lobanov/setup-xcode from 1.5.0 to 1.5.1 (#2936).
+  (contributed by @app/dependabot)
+
+- Build: Enable signing of macOS binary with self signed certificate. If a code signing certificate is set up but no notarization information given, Autobuild will now sign the builds assuming a self signed certificate is given (#2944).
+  (contributed by @ann0see)
+
+- Internal: Fix C++ build warnings regarding copy constructor and assignment functions (#2946).
+  (contributed by @hoffie)
+
+- Internal: Reduced permissions in Autobuild for security hardening (#2953).
+  (contributed by @ann0see)
+
+- Build: Updated bundled JACK (Windows-only) to version 1.9.21.20221111 (#2960).
+  (contributed by @app/github-actions)
+
+- Build: Updated bundled Qt6 to version 6.4.1 (#2961).
+  (contributed by @app/github-actions)
+
+- Build: Updated bundled Qt6 to version 6.4.2 (#2991).
+  (contributed by @app/github-actions)
+
+- Make checkboxes self-labelling (#2996).
+  (contributed by @pljones)
+
+- Translations updated from Hosted Weblate (#2998).
+  (contributed by @weblate)
+
+- Internal: The JACK build will now install directly from the JACK GitHub repository (#3003).
+  (contributed by @ann0see)
+
+- CONDENSE with https://github.com/jamulussoftware/jamulus/pull/2833 (#3008).
```

Comment:
Not sure what this means.  Maybe skip change log?

Reply:
Again, something for the last release. I think maybe we could automate this too. This should be added to another changelog entry.